editor/meta-boxes/meta-boxes-iframe/index.js

```diff
@@ -12,9 +13,12 @@ import { addQueryArgs } from '@wordpress/url';
 import { Component } from '@wordpress/element';
 
 /**
- * Internal dependencies.
+ * Internal dependencies
  */
-import MetaBoxPanel from './panel.js';
+import './style.scss';
+import './meta-box-iframe.scss';
+import { handleMetaBoxReload, metaBoxStateChanged } from '../../actions';
+import { getMetaBox, isSavingPost } from '../../selectors';
 
 class MetaBoxIframe extends Component {
```

Comment:
The class name is `MetaBoxIframe` but the folder name `meta-boxes-iframe`. We should be consistent one way or the other.